data/strings/types_strings.txt

```diff
@@ -11093,19 +11153,23 @@
 
   [type.landuse.residential.rural]
     en = landuse-residential-rural
+    pl = Tereny wiejskie
 
   [type.landuse.residential.urban]
     en = landuse-residential-urban
+    pl = Tereny miejskie
 
   [type.landuse.retail]
     en = landuse-retail
     pl = Obszar handlu
 
   [type.landuse.salt_pond]
     en = landuse-salt_pond
+    pl = Słone jezioro
 
   [type.landuse.village_green]
     en = landuse-village_green
+    pl = Nawsie
```

Comment:
are you sure?
https://pl.wikipedia.org/wiki/Nawsie

Reply:
Yes, in this issue "Nawsie" is correct. Another meaning of this word: https://pl.m.wikipedia.org/wiki/Nawsie_(urbanistyka)
Also check
https://wiki.openstreetmap.org/wiki/Pl:Tag:landuse%3Dvillage_green

Reply:
Looks like a niche term but ok.

Reply:
"Nawsie" is not popular Polish word, you are right :-)